mlflow/models/evaluation/default_evaluator.py

```diff
@@ -944,7 +950,9 @@ def _evaluate_classifier(self):
                 lambda: plot_lift_curve(self.y, self.y_probs),
                 "lift_curve_plot",
             )
-
+        # Skip confusion matrix for baseline model evaluation
+        if self.is_baseline_model:
+            return self._log_and_return_evaluation_result()
```

Comment:
Does `_log_and_return_evaluation_result()` still log an EvaluationResult instance to MLflow? If so, we probably don't want to log baseline results to MLflow.